.github/workflows/publish-layoutlib.yml

```
@@ -0,0 +1,35 @@
+name: publish-layoutlib
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
```

```suggestion
on:
  push:
    tags:
      - '**'
```

Nit: Might be better to copy release.yml to use tag-based releases.  Actually, no, that might conflict with the normal release process.  Maybe limit line 8 to use a special branch regex, i.e., something like `**/layoutlib**`.

---

yes, sorry something is up with github. I updated this with a `layoutlib` regex but it seems github isn't updating although the code is pushed 🤔

---

@JakeWharton any ideas?

---

https://github.com/cashapp/paparazzi/compare/master...gpowell/publish-layoutlib
shows 2 commits.

This pr only has my initial commit?

---

nevermind, it just updated. Must have been a weird Github PR update delay IDK

---

Yeah, there was an incident around this time.